<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>矿山微震监控系统 </string>
  </property>
  <property name="windowIcon">
   <iconset resource="myicons.qrc">
    <normaloff>:/image/images/lnu.jpg</normaloff>:/image/images/lnu.jpg</iconset>
  </property>
  <property name="toolTip">
   <string extracomment="矿高频微震地面台网监控系统"/>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(0, 0, 255);
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="locale">
        <locale language="Chinese" country="China"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>XX矿高频微震地面台网监控系统-辽宁大学</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Adobe 楷体 Std R</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>运行(&amp;R)</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="dailyStatement">
       <property name="font">
        <font>
         <family>Adobe 楷体 Std R</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>日报表(&amp;D)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="monthlyStatement">
       <property name="font">
        <font>
         <family>Adobe 楷体 Std R</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>月报表(&amp;M)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="font">
        <font>
         <family>Adobe 楷体 Std R</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>帮助(&amp;H)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>700</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="zoomIn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>放大(&amp;+)</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="zoomOut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>缩小(&amp;-)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="restoreButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>还原(&amp;B)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lastestIncidentLabel">
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>最新事件：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDockWidget" name="dockWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="floating">
          <bool>false</bool>
         </property>
         <property name="features">
          <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
         </property>
         <property name="allowedAreas">
          <set>Qt::AllDockWidgetAreas</set>
         </property>
         <widget class="QWidget" name="dockWidgetContents">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <widget class="QAxWidget" name="axWidget">
           <property name="control" stdset="0">
            <string>{74a777f8-7a8f-4e7c-af47-7074828086e2}</string>
           </property>
           <property name="sizePolicy" stdset="0">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops" stdset="0">
            <bool>true</bool>
           </property>
           <property name="statusTip" stdset="0">
            <string/>
           </property>
           <property name="autoFillBackground" stdset="0">
            <bool>false</bool>
           </property>
           <property name="styleSheet" stdset="0">
            <string notr="true">QPushButton{
	background-color: rgb(255, 253, 229);
}</string>
           </property>
           <property name="AutoActive" stdset="0">
            <bool>true</bool>
           </property>
           <property name="AutoZoomAll" stdset="0">
            <bool>false</bool>
           </property>
           <property name="BrowseMode" stdset="0">
            <bool>false</bool>
           </property>
           <property name="ShowCommandWindow" stdset="0">
            <bool>false</bool>
           </property>
           <property name="ShowMenuBar" stdset="0">
            <bool>false</bool>
           </property>
           <property name="ShowModelBar" stdset="0">
            <bool>false</bool>
           </property>
           <property name="ShowStatusBar" stdset="0">
            <bool>false</bool>
           </property>
           <property name="ShowToolBars" stdset="0">
            <bool>false</bool>
           </property>
           <property name="Theme" stdset="0">
            <number>0</number>
           </property>
           <property name="geometry" stdset="0">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>600</width>
             <height>600</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="T1Button">
           <property name="text">
            <string>T1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T2Button">
           <property name="text">
            <string>T2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T3Button">
           <property name="text">
            <string>T3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T4Button">
           <property name="text">
            <string>T4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T5Button">
           <property name="text">
            <string>T5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="T6Button">
           <property name="text">
            <string>T6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T7Button">
           <property name="text">
            <string>T7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T8Button">
           <property name="text">
            <string>T8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="T9Button">
           <property name="text">
            <string>T9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="displayButton">
           <property name="text">
            <string>显示</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myicons.qrc"/>
 </resources>
 <connections/>
</ui>
